taxi_service/settings.py

```diff
@@ -118,12 +114,13 @@
 
 USE_TZ = True
 
-
 # Static files (CSS, JavaScript, Images)
 # https://docs.djangoproject.com/en/4.0/howto/static-files/
 
 STATIC_URL = "static/"
 
+STATICFILES_DIRS = [BASE_DIR / "static"]
```

Comment:
Ensure that the `STATICFILES_DIRS` setting is correctly configured to include the `static` directory. This is necessary for Django to serve static files properly.